src/boundarySolver/physics/sublimation.cpp

```diff
@@ -23,7 +23,8 @@ ablate::boundarySolver::physics::Sublimation::Sublimation(std::shared_ptr<subMod
       pressureGradientScaling(std::move(pressureGradientScaling)),
       radiation(std::move(radiationIn)),
       radiationInterval((intervalIn ? intervalIn : std::make_shared<io::interval::FixedInterval>())),
-      emissivity(emissivityIn == 0 ? 1.0 : emissivityIn),
+      emissivity(parametersIn ? parametersIn->Get<PetscReal>("emissivity", 1.) : 1.),
+      absorptivity(parametersIn ? (parametersIn->Get<PetscReal>("absorptivity", -1) == -1 ? parametersIn->Get<PetscReal>("emissivity", 1.) : parametersIn->Get<PetscReal>("absorptivity", -1.)) : 1.),
```

Comment:
Instead of doing the -1 double comparisons we may want to grab GetString from the parameters.  It returns an optional if available, so you can treat it as the left hand side of the conditional.

Reply:
That makes sense, that'll make it a bit more readable!  I'll make the change and get back to you. I'm also going to add in a 1DDiffusionMonitor that can be used with 1D diffusion flames for coupled boundaries that will just return the mass flux to a log gable if that makes sense to do.